client/src/pages/searches/Search.js

```diff
@@ -1570,6 +1581,27 @@ const Search = ({
   function closeSaveModal() {
     setShowSaveSearch(false)
   }
+
+  function getResultsLimit() {
+    const authorizationGroupUuids = [
+      appSettings[UNLIMITED_EXPORTS_AUTHORIZATION_GROUP]
```

Comment:
This setting does not contain a list, just one uuid, does it not? So why put it in an array?